backend/src/server.spec.ts

```diff
@@ -2,9 +2,13 @@ import supertest, { Response } from 'supertest';
 import mongoose from 'mongoose';
```

Comment:
You are not doing a server.spec.ts ... you have to create one spec file for each controller instead like todos.spec.ts and register.spec.ts